### PR TITLE
Update Lesson_3_Let_Users_Mint_Your_NFT.md

### PR DESCRIPTION
**Pull request by ponyolovesham**

### CODE REVIEW

File: JS_DAO/en/Section_2/Lesson_3_Let_Users_Mint_Your_NFT.md [hunk 1]
```diff
@@ -91,7 +91,7 @@ const App = () => {
 
   // The signer is required to sign transactions on the blockchain.
   // Without it we can only read data, not write.
-  const signer = provider ? provider.getSigner() : undefined;
+  const signer = provider && provider.getSigner();
```
Comment (ponyolovesham) [inline JS_DAO/en/Section_2/Lesson_3_Let_Users_Mint_Your_NFT.md:94]: @adilanchian looking at what `provider` could return in useWeb's source code:
```js
export interface Web3ContextInterface {
    error?: Error;
    chainId?: number;
    balance?: Balance;
    provider?: Web3Provider;
    connector?: AbstractConnector & {
        [key: string]: any;
    };
    activeProvider?: ExternalProvider;
    address?: string;
    connectors: ConnectorType[];
    connectWallet: ReturnType<typeof useConnectWallet>;
    disconnectWallet: () => void;
    getNetworkMetadata: (chainId: number) => NetworkMetadata;
}
```
`provider` is optional, so the `provider && provider.getSigner();` will return `undefined` if the returned `provider` from invoking `useWeb3`

File: JS_DAO/en/Section_2/Lesson_3_Let_Users_Mint_Your_NFT.md [hunk 2]
```diff
@@ -159,7 +159,7 @@ const App = () => {
       <h1>Mint your free 🍪DAO Membership NFT</h1>
       <button
         disabled={isClaiming}
-        onClick={() => mintNft()}
+        onClick={mintNft}
```
Comment (ponyolovesham) [inline JS_DAO/en/Section_2/Lesson_3_Let_Users_Mint_Your_NFT.md:162]: onClick={() => mintNft()} is not necessary, we have a named function already to pass in as is without invoking it

Comment (ponyolovesham) [inline JS_DAO/en/Section_2/Lesson_3_Let_Users_Mint_Your_NFT.md:162]: @adilanchian 
 onClick={() => mintNft()} is not necessary, we have a named function already to pass in as is without invoking it

**Comment by ponyolovesham** [comment]
> @ponyolovesham - can you describe a bit more about your changes here please?

hi! add comments in file changes lmk if you have any questions!

**Review by helloitsm3 — COMMENTED** [review]
PR is outdated. Content already contains fixed for this.

**Pull request closed without merging**